resources/fixtures/Cameo/Cameo-Auro-Spot-300.qxf

```diff
@@ -139,7 +139,7 @@
  <Channel Name="Prism">
   <Group Byte="0">Prism</Group>
   <Capability Min="0" Max="5" Preset="PrismEffectOff">Prism off</Capability>
-  <Capability Min="6" Max="10" Preset="PrismEffectOn" Res1="3">Prism in</Capability>
+  <Capability Min="6" Max="10" Preset="PrismEffectOn" Res1="0">Prism in</Capability>
```

Comment:
@kluszon This looks like a regression to me. `Preset="PrismEffectOn" Res1="3">Prism in` is correct.